.jshintrc

```diff
@@ -1,15 +1,40 @@
 {
-  "camelcase": true,
+  // Put curly braces around conditional blocks & loops
   "curly": true,
+
+  // Use deep-equals for comparison
   "eqeqeq": true,
+
+  // Prohibit extension of native objects, i.e., String
   "freeze": true,
-  "indent": 2,
-  "newcap": true,
-  "quotmark": "single",
+
+  // Set max depth of nested blocks
   "maxdepth": 3,
-  "maxstatements": 15,
-  "maxlen": 80,
+
+  // Set max number of statements allowed per function
+  "maxstatements": 25,
```

Comment:
Oh, 25! This might fit in the category of things-we-should-not-try-to-retrospectively-style...but also: why 25?

Reply:
Arbitrary. The existing one has 15, which seemed a little short for some of the Highcharts helper functions, which can get a bit lengthy... What do you think?

Reply:
I think 25 is okay....I see this warning as a strategy for helping us think about modular JS, i.e., if your function has 25 statements, maybe it's doing too many things.